starter-code/index.html

```diff
@@ -0,0 +1,110 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css" integrity="sha384-Zug+QiDoJOrZ5t4lssLdxGhVrurbmBWopoEl+M6BdEfwnCJZtKxi1KgxUyJq13dy" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+
+    <title>Home - revera</title>
+  </head>
+  <body>
+    <div class="hfeed site">
+
+        <header id="header" class="container">
+            <div class="row">
+                <div class="brand col-sm-4">
+                    <h1>Revera</h1>
+                </div>
+                <div class="nav col-sm-8">
+                    <div id="submenu">
+                        <ul class="sfmenu sf-shadow sf-js-enabled">
+                          <li class="menu-item"><a>Home</a></li>
+                          <li class="menu-item"><a>About</a></li>
+                          <li class="menu-item"><a>Sample page</a></li>
+                          <li class="menu-item"><a>Portfolio</a></li>
+                          <li class="menu-item"><a>Blog</a></li>
+                          <!-- <li class="menu-item"><a>Parent page</a></li>
+                          <li class="menu-item"><a>Contact</a></li> -->
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </header>
+        <!-- header  with nav-->
+
+        <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
+                <div class="carousel-inner">
+                  <div class="carousel-item active">
+                    <img class="d-block w-100 grayscale" src="http://demo.fabthemes.com/revera/files/2013/08/5824862885_d6ba1e611f_o-1280x550.jpg" alt="First slide">
+                  </div>
+                  <div class="carousel-item">
+                    <img class="d-block w-100 grayscale" src="http://demo.fabthemes.com/revera/files/2013/08/wallpaper-939317-1280x550.jpg" alt="Second slide">
+                  </div>
+                  <div class="carousel-item">
+                    <img class="d-block w-100 grayscale" src="http://demo.fabthemes.com/revera/files/2013/08/beautiful_poppies-wallpaper-1920x1080-1280x550.jpg" alt="Third slide">
+                  </div>
+                </div>
+                <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev">
```

Comment:
Arrows are not working for me. Bootstrap is also using jQuery, so you may have something broken related to this jquery.